composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "silverstripe/login-forms",
+    "description": "A collection of templates for the CMS login screens",
+    "type": "silverstripe-vendormodule",
+    "license": "BSD-3-Clause",
+    "homepage": "https://github.com/silverstripe/login-forms",
+    "keywords": [
+        "silverstripe",
+        "template",
+        "login",
+        "styling"
+    ],
+    "require": {
+        "silverstripe/framework": "^4"
+    },
+    "require-dev": {
```

Comment:
Can we add `"scriptfusion/phpunit-immediate-exception-printer"`? When the output doesn't run in to the hundreds of tests, it's very useful and visual:
https://circleci.com/gh/Firesphere/silverstripe-bootstrapmfa/117

Reply:
We generally try not to introduce any dependencies that aren't part of our standardised modules - if you'd like this to be introduced I think it'd be best to raise an RFC on silverstripe/framework

Reply:
@ScopeyNZ @robbieaverill @NightJar thoughts?

Reply:
I do locally ;) Which Is why I add it to my repos